internal/confmapprovider/discovery/bundle/bundle.d/receivers/redis.discovery.yaml.tmpl

```diff
@@ -0,0 +1,69 @@
+{{ receiver "redis" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)redis"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)redis" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)redis"
+  config:
+    default:
+      endpoint: '`endpoint`'
```

Comment:
I think this field should already be inserted by the receiver creator. Do you know if this works (if not I can address where necessary)?

```suggestion
```

Reply:
I was not able to get it to work without adding that in. 

When I remove the whole config section I get a `no receiver configuration specified in config` error.

Also erroring when just removing endpoint and leaving the `config` and `default` fields.